src/components/activities/RosterPanel.tsx

```diff
@@ -18,6 +20,20 @@ interface RosterPanelProps {
   onClick?: (participant: Participant) => void;
 }
 
+const findMember = async (orgId: string, memberId: string) => {
+  return (await apiFetch<{ data: MemberInfo }>(`/api/v1/organizations/${orgId}/members/${memberId}`)).data;
+};
+
+const formatPhoneNumber = (phoneNumberString: string, trimIntlCode?: boolean) => {
```

Comment:
Do we need `trimIntlCode`? If we need it, can we give it a default value so most callers can ignore it?

Reply:
refactored to explicitly default false